Src/WitsmlExplorer.Api/Services/TrajectoryService.cs

```diff
@@ -51,10 +51,17 @@ public async Task<List<TrajectoryStation>> GetTrajectoryStations(string wellUid,
                 Uid = tStation.Uid,
                 DTimStn = tStation.DTimStn,
                 TypeTrajStation = tStation.TypeTrajStation,
-                Md = (tStation.Md == null) ? null : new LengthMeasure { Uom = tStation.Md.Uom, Value = StringHelpers.ToDecimal(tStation.Md.Value) },
-                Tvd = (tStation.Tvd == null) ? null : new LengthMeasure { Uom = tStation.Tvd.Uom, Value = StringHelpers.ToDecimal(tStation.Tvd?.Value) },
-                Incl = (tStation.Incl == null) ? null : new LengthMeasure { Uom = tStation.Incl.Uom, Value = StringHelpers.ToDecimal(tStation.Incl?.Value) },
-                Azi = (tStation.Azi == null) ? null : new LengthMeasure { Uom = tStation.Azi.Uom, Value = StringHelpers.ToDecimal(tStation.Azi?.Value) }
+                Md = LengthMeasure.FromWitsml(tStation.Md),
+                Tvd = LengthMeasure.FromWitsml(tStation.Tvd),
+                Incl = LengthMeasure.FromWitsml(tStation.Incl),
+                Azi = LengthMeasure.FromWitsml(tStation.Azi),
+                Dls =  LengthMeasure.FromWitsml(tStation.Dls),
+                Mtf =  LengthMeasure.FromWitsml(tStation.Mtf),
+                Gtf =  LengthMeasure.FromWitsml(tStation.Gtf),
+                DispNs =  LengthMeasure.FromWitsml(tStation.DispNs),
+                DispEw =  LengthMeasure.FromWitsml(tStation.DispEw),
+                VertSect = LengthMeasure.FromWitsml(tStation.VertSect)
```

Comment:
A lot more properties are listed in the comment in the PR. Any reason for why you did not add all of them?

Reply:
The missing properties were added as viewable in the update trajectory station modal.